src/ngraph/op/experimental/quantized_conv.cpp

```diff
@@ -40,15 +40,17 @@ op::QuantizedConvolution::QuantizedConvolution(const shared_ptr<Node>& data_batc
     , m_data_dilation_strides(data_dilation_strides)
     , m_requantize(requantize)
 {
-    constructor_validate_and_infer_types();
```

Comment:
If possible, please put this back in. Otherwise, please add a comment on why we are skipping it.